src/ia/js/IAPage.js

```diff
@@ -242,14 +261,26 @@
             'other_queries'
         ],
 
+        updateData: function(ia_data, x, edited) {
+            $.each(ia_data.live, function(key, value) {
+                if (edited) {
+                    x[key] = ia_data.edited[key]? ia_data.edited[key] : value;
+                } else {
+                    x[key] = value;
```

Comment:
Is it possible to just return `ia_data.live` if `edited` is `false` so that we don't have to go through the whole thing again? Not sure if I'm understanding it correctly, but it looks like `ia_data.live` is the same as `x` when `edited` is `false`.